src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -63,6 +65,10 @@
 public final class RqMultipartTest {
 
     /**
+     * T2 temporary part name.
+     */
+    private static final String PART_T2 = "t2";
```

Comment:
@raphaelln please convert it into local variable instead. It should be limited to scope of just one test